test/e2e/azure_test.go

```diff
@@ -76,6 +80,17 @@ var _ = Describe("Workload cluster creation", func() {
 		namespace, cancelWatches, err = setupSpecNamespace(ctx, clusterNamePrefix, bootstrapClusterProxy, artifactFolder)
 		Expect(err).NotTo(HaveOccurred())
 
+		By("Initialize bootstrap client to install add-ons")
```

Comment:
Would it be feasible to detect the CAPI version here and somehow fall back to the old `helm install` behavior if it's not >= 1.5.0?